contracts/subsidy-types/README.md

```diff
@@ -0,0 +1,3 @@
+# subsidy-types
+
+This module provides types which are needed for the interaction with any contract which supports subsidy.
```

Comment:
This whole crate is not clear to me.
I understand the subsidy flow, but cannot see a real use case right now (apart from tests)

Reply:
good use case could be for example for the bridge - a bridge will likely need it

Reply:
removed subsidy-types crate